aten/src/ATen/native/quantized/cpu/qnnpack/src/q8gemm/8x8-dq-aarch64-neon.S

```diff
@@ -45,6 +45,11 @@ BEGIN_FUNCTION pytorch_q8gemm_dq_ukernel_8x8__aarch64_neon
     # Load multiplier
     LD1R {v26.4s}, [x8]
 
+    # v8 := zero
+    EOR v8.16b, v8.16b, v8.16b
```

Comment:
Not clear what was the bug here. I see v8 and v9 getting overwritten elsewhere, so I am not sure of the need to zero these guys.

Reply:
SMLAL is a (signed long) multiply-accumulate instruction.  v8 and v9 are used as accumulators there and should have been initialized to zero.